hpx/parallel/datapar/execution_policy.hpp

```diff
@@ -24,18 +24,18 @@
 #include <type_traits>
 #include <utility>
 
-namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v1)
+namespace hpx { namespace parallel { namespace execution
```

Comment:
Ok. I clearly miss where the versioning is done here. I miss the inline namespace v2 or something similar.